udtube/callbacks.py

```diff
@@ -101,13 +102,22 @@ def _fill_in_tags(
             attr (str): attribute on tokens where the tags should be inserted.
             tags (Iterator[str]): tags to insert.
         """
-        # Note that in when MWEs are present, the iterators with predicted tags
+        # Note that when MWEs are present, the iterators with predicted tags
```

Comment:
period at end of comment sentence please

Reply:
What? This has a period two lines down and it's also not a complete sentence on its own. Was this a mistake?

Reply:
Oh, nevermind, I got punk'd by the display here. Resolving.